lib/kubeclient.rb

```diff
@@ -13,6 +13,9 @@ module Kubeclient
   class Client < Common::Client
     attr_reader :api_endpoint
 
+    SERVICE_ACCOUNT_TOKEN_FILE =
```

Comment:
Please do not hardcode this in the client. Passing the path to file to the client should do the work.
Hardcoding it means duplicating server side configuration/logic, is easily breakable by server side changes and will require more maintenance.